feature-extensions/multiaccount/README.md

```diff
@@ -0,0 +1,102 @@
+- Installer will use the AWS credentials that you have configured in your system by running 'aws configure'
+
+## Basic Usage
+
+```sh
+cd JAZZ_INSTALLER_DIRECTORY/feature-extensions/multiaccount
+python setup.py [-h] [--aws-region AWS_REGION]
+                [--jazz-stackprefix JAZZ_STACKPREFIX] [--aws-accesskey AWS_ACCESSKEY]
+                [--aws-secretkey AWS_SECRETKEY] [--jenkins-url JENKINS_URL]
+                [--jenkins-username JENKINS_USERNAME]
+                [--jenkins-password JENKINS_PASSWORD]
+                [--jazz-username JAZZ_USERNAME]
+                [--jazz-password JAZZ_PASSWORD]
+                [--jazz-apiendpoint JAZZ_APIENDPOINT]
+                {install} ...
+```
+Note:
+   - JAZZ_INSTALLER_DIRECTORY - Directory path of jazz installation
+   - Install option will ask user for raw inputs for all the missing arguments (similar to below)
+```sh
+    Please enter the environment prefix you used for your Jazz install:
+    Enter AWS accesskey of the new account:
+    Enter secretkey of the new account:
+    Enter AWS regions with space delimiter:
+    Please enter the Jenkins URL(without http):
+    Please enter the Jenkins Username:
+    Please enter the Jenkins Password:
+    Please enter the Jazz Admin Username:
+    Please enter the Jazz Admin Password:
+    Please enter the Jazz API Endpoint(Full URL):
+```
+
+## Example (Install option)
+
+### With all the parameters as arguments
+
+```sh
+cd JAZZ_INSTALLER_DIRECTORY/feature-extensions/multiaccount
+python setup.py --aws-region us-east-1 us-west-2
+                --jazz-stackprefix STACKPREFIX
+                --aws-accesskey XXXXX
+                --aws-secretkey XXXXX
+                --jenkins-url JENKINSURL
+                --jenkins-username JENKINSUSER
+                --jenkins-password JENKINSPASSWORD
+                --jazz-username JAZZADMIN
+                --jazz-password JAZZPASSWORD
+                --jazz-apiendpoint JAZZ_APIENDPOINT install
+
+```
+
+### With missing parameters as arguments ###
+
+```sh
+cd JAZZ_INSTALLER_DIRECTORY/feature-extensions/multiaccount
+python setup.py install
+
+# Enter the missing input params
+Please enter the environment prefix you used for your Jazz install:
+Enter AWS accesskey of the new account:
+Enter secretkey of the new account:
+Enter AWS regions with space delimiter:
+Please enter the Jenkins URL(without http):
+Please enter the Jenkins Username:
+Please enter the Jenkins Password:
+Please enter the Jazz Admin Username:
+Please enter the Jazz Admin Password:
+Please enter the Jazz API Endpoint(Full URL):
+```
+
+## Example (Uninstall option)
+
+### With all the parameters as arguments
+
+```sh
+cd JAZZ_INSTALLER_DIRECTORY/feature-extensions/multiaccount
+python triggerJenkinsDeleteResources.py --jenkins-url JENKINSURL
+                                        --jenkins-username JENKINSUSER
+                                        --jenkins-password JENKINSPASSWORD --account-details all
```

Comment:
`--account-details all` - Would this delete all resources in 
- one secondary account (& all regions within it)? 
- one secondary account (one region within it)?
- all secondary accounts (excluding master account)?
- all secondary accounts (including master account)? - may be not!

Reply:
all secondary accounts (including master account's extra regions, ie excluding default region)